content/releases.md

```diff
@@ -1,4 +1,89 @@
-## v1.1.0 - "Training Wheels"
+## Release 1.2.0 - "Kubernetes For The Edge"
+
+[Get Started with release 1.2.0](/docs/1.1.0/getting-started/core-concepts.html)
+
+Large ioFog release brings a set of features for seamless integration of Kubernetes and Edge Computing. The release
+comes with a new command line interface for Edge Compute Network management called _iofogctl_ and other supplementary
+tools, such as _platform tools_ for cluster and infrastructure management.
+
+### What's New?
+
+- **Kubernetes integration!** First release of ioFog supporting Kubernetes integration, and first releases of [iofog-kubelet](https://github.com/eclipse-iofog/iofog-kubelet/releases/tag/v1.2.0) and [iofog-operator](https://github.com/eclipse-iofog/iofog-operator/releases/tag/v1.2.0) microservices
+- **[Iofogctl](https://github.com/eclipse-iofog/iofogctl/releases/tag/v1.2.0)** command line interface for management of Edge Compute Networks (see [iofogctl tutorial](/docs/1.2.0/tools/iofogctl.html)) running on both Linux and Mac
+- **[Helm chart](https://github.com/eclipse-iofog/helm/releases/tag/v1.2.0)** for easy installation of ioFog Edge Compute Network to existing Kubernetes cluster (see [helm tutorial](/docs/1.2.0/getting-started/how-to-helm.html))
+- **[Platform tools](https://github.com/eclipse-iofog/platform/tree/1.2.0)** for easy infrastructure and Kubernetes cluster setup on Google Cloud Platform and Packet (see [platform tutorial](/docs/1.2.0/tools/platform-tools.html))
+- Many bugfixes in the ioFog engine: [controller](https://github.com/eclipse-iofog/Controller/releases/tag/v1.2.0), [connector](https://github.com/eclipse-iofog/Connector/releases/tag/v1.2.0) and [agent](https://github.com/eclipse-iofog/Agent/releases/tag/v1.2.0)
+
+### Changelogs
+
+##### Agent (v1.2.0)
+
+* Send Agent's external IP to Controller
+* Bugfix: Selecting the network interface that has Controller connectivity
+* Bugfix: Use local docker images when offline
+
+##### Connector (v1.2.0)
+
+* Limit port range to 50 ports in default configuration file
+
+##### Controller (v1.2.0)
+
+* Return Agent's external IP for Kubelet
+* Add uptime to status endpoint
+* Bugfix: Requests not failing if with additional properties
+
+##### Kubelet (v1.2.0)
+
+* Initial release!
+
+##### Iofogctl (v1.2.0)
+
+* Initial release!
+
+##### Helm Chart (v1.2.0)
+
+* Initial release!
+
+##### Platform Tools (v1.2.0)
+
+* Initial release!
+
+##### Quickstart + Tutorial (v1.2.0)
+
+* 
```

Comment:
Section will already be covered, maybe just give a link to the updated tutorial section, so if anyone comes into this seeing release notes first, easy link to find where to get started?